src/genie/libs/parser/iosxe/show_isis.py

```diff
@@ -503,7 +503,14 @@ def cli(self, output=None):
                     elif mtype == 'IS':
                         is_dict = lsp_dict.setdefault('is_neighbor', {}).setdefault(ip, {})
 
-                    is_dict.update({'neighbor_id': ip,
+                    if 'metric' in is_dict.keys():
+                        if type(is_dict['metric']) is list:
+                            is_dict['metric'].append(int(group['metric']))
+                        elif type(is_dict['metric']) is int:
```

Comment:
this is minor, but I think it would be better to use isinstance instead of type

```suggestion
                        if isinstance(is_dict['metric'], list):
                            is_dict['metric'].append(int(group['metric']))
                        elif isinstance(is_dict['metric'], int):
```